youtube_dl/extractor/extractors.py

```diff
@@ -247,6 +247,9 @@
     CNNArticleIE,
 )
 from .coub import CoubIE
+from .cozytv import (
+    CozyTVReplayIE,
+)
```

Comment:
Whereas, if there's just one item (and you're not planning to add another extractor), this can be one line:
```suggestion
from .cozytv import CozyTVReplayIE,
```